pkg/apis/policy/validation/validation.go

```diff
@@ -307,21 +304,76 @@ func IsValidSysctlPattern(name string) bool {
 	return sysctlPatternRegexp.MatchString(name)
 }
 
+func validatePodSecurityPolicySysctlListsDoNotOverlap(allowedSysctlsFldPath, forbiddenSysctlsFldPath *field.Path, allowedUnsafeSysctls, forbiddenSysctls []string) field.ErrorList {
+	allErrs := field.ErrorList{}
+	for i, allowedSysctl := range allowedUnsafeSysctls {
+		isAllowedSysctlPattern := false
+		allowedSysctlPrefix := ""
+		if strings.HasSuffix(allowedSysctl, "*") {
+			isAllowedSysctlPattern = true
+			allowedSysctlPrefix = strings.TrimSuffix(allowedSysctl, "*")
+		}
+		for _, forbiddenSysctl := range forbiddenSysctls {
```

Comment:
`forbiddenSysctlsFldPath.Index(i)` and `forbiddenSysctls[i]` below should be using `j` index saved here, not `i`

Reply:
Copy-paste thing, shame on me. Fixed